src/projects-card/ProjectsCard.css

```diff
@@ -0,0 +1,23 @@
+.mantine-Card-root {
+    flex-shrink: 0;
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+    border-radius: 30px;
```

Comment:
minor note but it seems like the bottom corners of the image are square 
<img width="593" alt="image" src="https://github.com/uoftblueprint/website-v2/assets/50971945/adba6731-da18-485c-9bb6-9ff84b516a08">

Reply:
Good catch!